core-chart/templates/postcreatehooks/transport-controller.yaml

```diff
@@ -124,7 +124,7 @@ spec:
           - name: setup-its-kubeconfig
             image: quay.io/kubestellar/kubectl:{{.Values.KUBECTL_VERSION}}
             imagePullPolicy: IfNotPresent
-            command: [ "bin/sh", "-c", "sh /mnt/config/get-kubeconfig.sh '{{.ITSName}}' true | base64 -d > /mnt/shared/transport-kubeconfig" ]
+            command: [ "bin/sh", "-c", "sh /mnt/config/get-kubeconfig.sh '{{ .ITSName }}' true | base64 -d > /mnt/shared/transport-kubeconfig" ]
```

Comment:
You just need to change this to make it work like the WDS case did before this PR.